src/components/pages/Donate/RenderDonatePage.jsx

```diff
@@ -0,0 +1,77 @@
+// The Donate page content goes here
+import React from 'react';
+import { Layout, Menu, Breadcrumb } from 'antd';
+import {
+  UserOutlined,
+  LaptopOutlined,
+  NotificationOutlined,
+} from '@ant-design/icons';
```

Comment:
oh you beat me to it! I'm still figuring it out. Those design templates from antd look really cool